docs/manual/src/docs/asciidoc/_includes/reactive/x509.adoc

```diff
@@ -0,0 +1,24 @@
+[[reactive-x509]]
+= Reactive X.509 Authentication
+
+Similar to <<x509,Servlet X.509 authentication>>, reactive x509 authentication filter allows to extract an authentication token from a certificate provided by a client.
+
```

Comment:
I'd like a minimal example configuration shown first with just `.x509()` then document all the properties (i.e. adding `authenticationManager` and `principalExtractor` in a separate configuration). This allows users to not only see the minimum requirements but then see all the options.

Reply:
I've added another example of x509 configuration to demonstrate how to change `authenticationManager` and `principalExtractor` and described how defaults will work.